### PR TITLE
[Docs] Fix DocC warnings in FirebaseAuth

### PR DESCRIPTION
**Pull request by ncooke3**
#no-changelog

### CODE REVIEW

**Review by morganchen12 — APPROVED** [review]

File: FirebaseAuth/Sources/Swift/AuthProvider/FederatedAuthProvider.swift [hunk 1]
```diff
@@ -22,6 +22,9 @@ import Foundation
     /// Used to obtain an auth credential via a mobile web flow.
     /// This method is available on iOS only.
     /// - Parameter uiDelegate: An optional UI delegate used to present the mobile web flow.
+    /// - Parameter completionHandler: Optionally; a block which is invoked asynchronously on the
+    /// main
+    /// thread when the mobile web flow is completed.
```
Comment (andrewheard) [inline FirebaseAuth/Sources/Swift/AuthProvider/FederatedAuthProvider.swift:27]: nit: Formatting

File: FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift [hunk 2]
```diff
@@ -339,6 +337,9 @@ import Foundation
     /// Used to obtain an auth credential via a mobile web flow.
     /// This method is available on iOS only.
     /// - Parameter uiDelegate: An optional UI delegate used to present the mobile web flow.
+    /// - Parameter completionHandler: Optionally; a block which is invoked asynchronously on the
+    /// main
+    /// thread when the mobile web flow is completed.
```
Comment (andrewheard) [inline FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift:342]: nit: Formatting

File: FirebaseAuth/Sources/Swift/AuthProvider/FederatedAuthProvider.swift [hunk 3]
```diff
@@ -22,6 +22,9 @@ import Foundation
...
     @available(iOS 13, tvOS 13, macOS 10.15, watchOS 8, *)
     @objc(getCredentialWithUIDelegate:completion:)
```
Comment (ncooke3) [inline FirebaseAuth/Sources/Swift/AuthProvider/FederatedAuthProvider.swift:29]: it's a bit unexpected but Apple generates the parameter name to be `completionHandler`, not `completion`:

```objc
getCredentialWithUIDelegate: /* type */uiDelegate completion:/* type */completion>>Handler<<
```

**Review by andrewheard — APPROVED** [review]

File: FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift (hunk 2 above)
Comment (ncooke3) [reply FirebaseAuth/Sources/Swift/AuthProvider/OAuthProvider.swift]: done.

File: FirebaseAuth/Sources/Swift/AuthProvider/FederatedAuthProvider.swift (hunk 1 above)
Comment (ncooke3) [reply FirebaseAuth/Sources/Swift/AuthProvider/FederatedAuthProvider.swift]: done.

**Pull request merged**